docs/current_docs/faq.mdx

```diff
@@ -94,7 +94,18 @@ It’s possible to send Traces by setting the `CI=true` variable in Dagger's run
 
 ### What language SDKs are available for Dagger?
 
-We have three types of SDKs, with varying levels of parity and support: official, community and experimental. We currently offer official SDKs for Go, TypeScript SDK and Python. A community SDK is available for PHP.
+We have three types of SDKs, with varying levels of parity and support: official, community and experimental. We currently offer official SDKs for Go, TypeScript SDK and Python. A community SDK is available for PHP and Java.
```

Comment:
```suggestion
We have three types of SDKs, with varying levels of parity and support: official, community and experimental. We currently offer official SDKs for Go, TypeScript SDK and Python. A community SDK is available for PHP, and an experimental SDK is available for Java.
```

@mircubed we refer to the Java SDK as experimental in other places:
- https://docs.dagger.io/ai-agents/#architecture
- https://dagger.io/blog/java-sdk